Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -1987,7 +1987,9 @@ static bool getEventTypeFromWebEvent(const WebEvent& event, NSEventType& eventTy
             continue;
         if ([NSStringFromSelector(item.action) isEqualToString:@"openWithPreview"])
             openInPreviewIndex = i;
-        PDFContextMenuItem menuItem { String([item title]), !![item isEnabled], !![item isSeparatorItem], static_cast<int>([item state]), !![item action], i };
+        PDFContextMenuItem menuItem { String([item title]), static_cast<ContextMenuItemEnablement>(!![item isEnabled]),
+            static_cast<ContextMenuItemIsSeparator>(!![item isSeparatorItem]), static_cast<int>([item state]),
+            static_cast<ContextMenuItemHasAction>(!![item action]), i };
```

Comment:
Isn't this UB? We didn't map the enumeration cases to literal values at the enum declaration.

```suggestion
        PDFContextMenuItem menuItem { String([item title]), [item isEnabled] ? ContextMenuItemEnablement::Enabled : ContextMenuItemEnablement::Disabled, [item isSeparatorItem] ? ContextMenuItemIsSeparator::Yes : ContextMenuItemIsSeparator::No, static_cast<int>([item state]), [item action] ? ContextMenuItemHasAction::Yes : ContextMenuItemHasAction::No, i };
```

Reply:
I thought it was ok given the fixed underlying type but went ahead and added the values to the enum

Reply:
I don't think this was UB? That said, Abrar's proposal looks nicer IMO so we may as well adopt it.

Reply:
Changed and removed the values from the enum definitions